templates/d1-openapi/drizzle/local.ts

```diff
@@ -0,0 +1,48 @@
+import crypto from "node:crypto";
+import fs from "node:fs";
+import { unstable_readConfig } from "wrangler";
```

Comment:
ah man i remember seeing this in the wrangler source code a few months ago and thinking "we should use this" then completely forgot. good catch